model/userrating.js

```diff
@@ -0,0 +1,15 @@
+function UserRating(userid,userName,rating) {
+  this.userid = userid;
+  this.userName = userName;
+  this.rating = (typeof rating !== 'undefined') ? parseInt(rating) :1;
```

Comment:
no reason to use `typeof` any longer with modern javascript. Just do:
`this.rating = rating !== undefined ? parseInt(rating) : 1`